### ISSUE TITLE
multisig-rotation-in-third.sh error "ERR: invalid rotation, new key set unable to satisfy prior next signing threshold" unless participant keys rotated and queried

### ISSUE THREAD

**Issue by kentbull** [WebOfTrust/keripy#773]
### Version

v1.1.13

### Environment

Mac OS, Python 3.12, M1 Mac

### Expected behavior

Rotating a multisig AID in the `scripts/demo/basic/multisig-rotation-in-third.sh` script should run and complete successfully.

### Actual behavior

The `kli multisig rotate` calls in `scripts/demo/basic/multisig-rotation-in-third.sh` return errors as shown below:

```
ERR: invalid rotation, new key set unable to satisfy prior next signing threshold
```

### Steps to reproduce

1. Clone the KERIpy repository and vLEI repository: 
  `git clone https://github.com/WebOfTrust/keripy`
  `git clone https://github.com/WebOfTrust/vLEI`
2. In KERIpy source root switch to v1.1.13: 
    ```bash
    cd keripy
    git switch v1.1.13
    ```
4. In separate terminals in parallel:
    - In its own terminal session run the vLEI server
      ```bash
      cd vLEI
      vLEI-server -s ./schema/acdc -c ./samples/acdc/ -o ./samples/oobis/
      ```
    - In its own terminal session run the demo witnesses
      ```bash
      cd keripy/
      kli witness demo
      # should show the following output:
      Witness wan : BBilc4-L3tFUnfM_wJr4S4OJanAv_VmF_dJNN6vkf2Ha
      Witness wil : BLskRTInXnMxWaGqcpSyMgo0nYbalW99cGZESrz3zapM
      Witness wes : BIKKuvBwpmDVA4Ds-EpL5bt9OqPzWPja2LigFYZN2YfX
      Witness wit : BM35JN8XeJSEfpxopjn5jr7tAHCE5749f0OobhMLCorE
      Witness wub : BIj15u5V11bkbtAxMA7gcNJZcax-7TgaBMLsQnMHpYHP
      Witness wyz : BF2rZTW79z4IXocYRQnjjsOuvFUQv-ptCf8Yltd7PfsM
      ```
5. Run the `scripts/demo/basic/multisig-rotation-in-third.sh` script
    In a separate terminal session run the following:
    ```bash
    cd keripy/scripts/demo
    source demo-scripts.sh
    ./basic/multisig-rotation-in-third.sh
    ```

**Comment by 2byrds** [comment]
I'll add as a candidate to be considered for the top 10 https://github.com/WebOfTrust/keri/discussions/78

**Comment by pfeairheller** [comment]
Closed with #774

**Comment by kentbull** [comment]
With the latest 1.2.0-dev2 version I'm hitting a different problem with this script, that the waiting for signatures on the multisig interaction hang at the end and never finish. Is there something that multisig3 needs to do to take part in the interaction event?
